data/json/itemgroups/guns.json

```diff
@@ -159,6 +159,7 @@
       [ "fn_fal", 40],
       [ "hk_g3", 40 ],
       [ "hk_g36", 30 ],
+      [ "henry_big_boy", 30],
```

Comment:
That would put it at the same weight as the Leadworks futuristic .223 carbine. I used the Long .45 as a reference for balance. 20 might be better though.